includes/definitions/discovery/fxm.yaml

```diff
@@ -32,16 +34,17 @@ modules:
                     oid: upsOutputFrequency
                     num_oid: .1.3.6.1.4.1.7309.6.1.4.2.
                     index: 'upsOutputFrequency.{{ $index }}'
-                    divisor: 10
                     descr: Output
+                    divisor: 10
```

Comment:
This has just moved a couple of lines, any reason?

Reply:
I just noticed the apc.yaml had the divisors after the descr line.

Reply:
The order doesn't matter.